### PR TITLE
Updates for Jakarta EE.

### PR DESCRIPTION
**Pull request by bshannon**
A first pass at updating the content for Jakarta EE.
There's still more work to be done.

### CODE REVIEW

File: CompatibilityRequirements.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ use of the previous version of the specification. However if it is
 considered that there will be no or few such implementations, or if
 they would always be provided as an add-on by the developers of vendor
 implementations, then this requirement may be relaxed after careful
-consideration by the specification's expert group.
+consideration by the specification project team.
```
Comment (erik-brangs) [inline CompatibilityRequirements.md]:
```suggestion
consideration by the specification project.
```

In most cases, you changed "specification's expert group" to "specification project". You could remove the "team" here too, for consistency.

File: CompatibilityRequirements.md [hunk 2]
```diff
@@ -126,18 +125,22 @@ vendor."
...
 interfaces or methods.
 
+**XXX** - This section needs to be updated with new rules for the actual
+removal of APIs and specifications from the Jakarta EE platform as anticipated
+in Jakarta EE 9, similar to what's done for the Java SE platform.
+
 ## Deprecated methods
 
-**6.** An expert group may decide to designate a method or interface as
+**6.** A specificaiton project may decide to designate a method or interface as
```
Comment (erik-brangs) [inline CompatibilityRequirements.md]:
```suggestion
**6.** A specification project may decide to designate a method or interface as
```

typo: "specificaiton" -> "specification"

File: CompatibilityRequirements.md [hunk 3]
```diff
@@ -184,8 +187,8 @@ were a listener or callback interface) then this is never permissible
 will be no or few wrapper implementations of the interface, or if they
 would always be provided as an add-on by the developers of vendor
 implementations, it may be permissible to add a new method to an
-existing interface after careful consideration by the specification's
-expert group  (see Rule 3 for a full description).
+existing interface after careful consideration by the specification
+project team  (see Rule 3 for a full description).
```
Comment (erik-brangs) [inline CompatibilityRequirements.md]:
```suggestion
project  (see Rule 3 for a full description).
```

As above: In most cases, you changed "specification's expert group" to "specification project".

File: CompatibilityRequirements.md [hunk 4]
```diff
@@ -197,18 +200,18 @@ No, because this would violates rules 2 and 3 above.
 Where behavior was undefined or optional in the previous version of the
 specification then it may be permissible to define more explicit
 behavior in a later version.  In deciding whether such a change is
-permissible, the expert group must give consideration to the behavior
-of existing implementations, especially the reference implementation.
+permissible, the specification project must give consideration to the behavior
+of existing implementations.
 Just because a particular behavior wasn't specified doesn't mean
-applications haven't come to depend on it.  Expert groups should strive
+applications haven't come to depend on it.  Specifcation projects should strive
```
Comment (erik-brangs) [inline CompatibilityRequirements.md]:
```suggestion
applications haven't come to depend on it.  Specification projects should strive
```
typo: "Specifcation" -> "Specification"

File: Specifications.md [hunk 5]
```diff
@@ -1,55 +1,49 @@
...
+| [Jakarta Batch](https://github.com/eclipse-ee4j/batch-api) | [1.0](https://jakarta.ee/specifications/batch/1.0/) |  |
+| [Jakarta Bean Validation](https://github.com/eclipse-ee4j/beanvalidation-spec) | [2.0](https://jakarta.ee/specifications/bean-validation/2.0/) |  |
+| [Jakarta Concurrency](https://github.com/eclipse-ee4j/concurrency-api) | [1.1](https://jakarta.ee/specifications/concurrency/1.1/) | [Eclipse Glassfish 5.1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads) |
+| [Jakarta Connectors](https://github.com/eclipse-ee4j/jca-api) | [1.7](https://jakarta.ee/specifications/connectors/1.7/) | [Eclipse Glassfish 5.1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads) |
+| [Jakarta Contexts and Dependency Injection](https://github.com/eclipse-ee4j/cdi) | [2.0](https://jakarta.ee/specifications/cdi/2.0/) |  |
+| [Jakarta Debugging Support for Other Languages](https://github.com/eclipse-ee4j/jsp-api) | [1.0](https://jakarta.ee/specifications/debugging/1.0/) | [Eclipse GlassFish 5.1](https://eclipse-ee4j.github.io/glassfish/)  |
```
Comment (erik-brangs) [inline Specifications.md]: The links for "Jakarta Debugging Support for Other Languages" goes to the JSP API. Is this correct?

Comment (bshannon) [reply Specifications.md]: Yes, the spec is part of that repo:
https://github.com/eclipse-ee4j/jsp-api/tree/master/jdsol-spec

File: CompatibilityRequirements.md (hunk 1 above)
Comment (bshannon) [reply CompatibilityRequirements.md]: Thanks for the review!  I fixed this and all your other comments.

**Pull request merged**